types.cc

```diff
@@ -749,6 +749,20 @@ void write_collection_value(managed_bytes_mutable_view& out, cql_serialization_f
     write_fragmented(out, single_fragmented_view(val));
 }
 
+void write_collection_value(managed_bytes_mutable_view& out, cql_serialization_format sf, const managed_bytes_view& val) {
```

Comment:
I don't understand this. It isn't used in this patch, at least not directly. Shouldn't it be part of the previous patch?

Reply:
It should. Moved to the previous patch. (It's only used later, so it doesn't really matter anyway though.)